test/gtest/ucs/test_rcache.cc

```diff
@@ -287,6 +296,43 @@ UCS_MT_TEST_F(test_rcache, get_unmapped, 6) {
     free(ptr);
 }
 
+UCS_MT_TEST_F(test_rcache, put_and_invalidate, 1)
```

Comment:
can u pls add short comment what the test is doing?

Reply:
added description